src/app/components/Pagination/Pagination.jsx

```diff
@@ -27,10 +27,11 @@ class Pagination extends React.Component {
     const intPage = parseInt(page, 10);
     const pageNum = type === 'Next' ? intPage + 1 : intPage - 1;
     const svg = type === 'Next' ? <RightArrowIcon /> : <LeftArrowIcon />;
+    const url = this.props.createAPIQuery({ page: pageNum });
```

Comment:
`/search?q=&page=2` is the desired result because the empty search keyword is intended so `q=` is a blank search keyword.

If you check out this page http://dev-discovery.nypl.org/bib/b12403270, the pagination button will correctly have `<a href="#" ...` so the `url ? ...` check is working properly, which means that `url` can be undefined.